src/components/HURUmap/Tutorial/Line.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+function Line({ start, end, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div key={start.x1 + start.y1} className={classes.lineContainer}>
+      <svg
+        key={start.x1 + start.y1}
```

Comment:
Instead of setting key on `div` and `svg` here, can't we set `key` on `Line` in `Connector`?